reactor-core/src/test/java/reactor/core/publisher/EmitterProcessorTest.java

```diff
@@ -906,7 +907,7 @@ public void tryEmitNextWithNoSubscriberFailsOnlyIfNoCapacity() {
 
 		StepVerifier.create(emitterProcessor)
 		            .expectNext(1)
-		            .then(emitterProcessor::emitComplete)
+		            .then(emitterProcessor::tryEmitComplete)
```

Comment:
isn't it a behaviour change of the test?

Reply:
not really, since `emitComplete` doesn't do anything in particular

Reply:
changed in latest commit